src/Couchbase.Lite.Shared/API/Sync/ReplicatorConfiguration.cs

```diff
@@ -193,6 +193,17 @@ public ReplicatorType ReplicatorType
         [NotNull]
         public IEndpoint Target { get; }
 
+        /// <summary>
+        /// A bool value indicating the delta sync is enabled or not. The
+        /// default value it true.
+        /// </summary>
+        [CanBeNull]
+        public bool DeltaSyncEnabled
+        {
+            get => !Options.NoDeltas;
+            set => _freezer.PerformAction(() => Options.NoDeltas = !value);
+        }
```

Comment:
@borrrden yes, DeltaSyncEnabled default value should be true. Not NoDelta is Delta.. On LiteCore, Jens uses "noDelta" as flag, but Pasin uses DeltaSyncEnabled on Lite.

Reply:
Oh, I see!  Things are flipped around here.